pkg/multicluster/razee/razee_cluster_manager.go

```diff
@@ -316,35 +319,57 @@ func channelName(cluster string, name string) string {
 	return "m4d.ibm.com" + "-" + cluster + "-" + name
 }
 
-func NewRazeeManager(url string, login string, password string, orgID string) multicluster.ClusterManager {
-	localAuth := &LocalAuthClient{
-		url:      url,
-		login:    login,
-		password: password,
-		token:    "",
+func NewRazeeManager(url string, login string, password string) (multicluster.ClusterManager, error) {
+	localAuth, err := local.NewClient(url, login, password)
+	if err != nil {
+		return nil, err
 	}
```

Comment:
@froesef I'm not sure why this code is marked as changed as its already in master.